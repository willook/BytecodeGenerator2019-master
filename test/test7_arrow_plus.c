void main () {
	int t = 33;
    int a = 5;

	t -> + 2 -> + 5;
	_print(t);

    a -> + 5;
    t -> + a;
    t -> + t;
    _print(t);
    _print(a);
}
