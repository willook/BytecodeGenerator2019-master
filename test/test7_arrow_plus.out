40
100
10
